### DISCUSSION TITLE
Uno Platform Solution Templates for Visual Studio 2022?

### DISCUSSION THREAD

**Discussion by gaetandezeiraud** [unoplatform/uno#7890] [Q&A] [answered]
How to install the Uno Platform Solution Templates on Visual Studio 2022? The one in the documentation only support Visual Studio 2017 and 2019.

**Comment by jeromelaban** [comment, marked as answer]
Please see this page: https://platform.uno/docs/articles/get-started-vs-2022.html

**Reply by gaetandezeiraud** [reply]
Thanks!
